robots.txt

```diff
@@ -0,0 +1,104 @@
+User-agent: *
+Disabllow: /checkacc.php*
+
+User-agent: MJ12bot
+Disallow: /
+
+User-agent: Mediapartners-Google*
+Disallow: /
+
+User-agent: IsraBot
+Disallow:
```

Comment:
Shall this line be `Disallow: /`? Missing "/".

The same comment applies to line 14.